CMakeLists.txt

```diff
@@ -3,19 +3,22 @@ cmake_minimum_required(VERSION 2.8.9)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 
+##
+##  GNU standard paths
+##
+include(GNUInstallDirs)
+
+set(PROJECT_VERSION "0.2")
 set(PACKAGE ${CMAKE_PROJECT_NAME})
 set(SERVICE_LIB ${PACKAGE})
 set(SERVICE_EXEC "${PACKAGE}-service")
 
-##
-##  GNU standard paths
-## 
+option(enable_tests "Build the package's automatic tests." ON)
+option(enable_coverage "Generate code coverage reports." OFF)
```

Comment:
This is not right, coverage reports are enabled with CMAKE_BUILD_TYPE=coverage, this would mean we need to set this twice. see https://gitlab.com/ubports/core/cmake-extras/-/blob/master/src/CoverageReport/EnableCoverageReport.cmake#L67